RELEASE.md

```diff
@@ -150,7 +150,8 @@ Once the workflow job is done, ensure all publication sources have the new artif
 ### 5. Download artifacts
 
 Once the full publish-ci.yml worflow is done, the release artifacts can be downloaded from the GitHub Artifacts (located in the "Summary" tab of the action).
-These are currently manual steps taken from the browser.
+Similarly, release artifacts can be downloaded from the docs-ci.yml workflow.
```

Comment:
I would add a new last section under `## Artifacts` that describes the documentation artifacts.

I do wonder about the javadoc ones though, b/c they are very big...

Reply:
Can you comment on the size of the other documentation artifacts? I could be in favor of only adding javadoc workflow artifact on release - would want to specifically note it in the yml file.

Reply:
Below are compressed file sizes:

Docs Artifacts
- 2.4MB  pyserver-docs-0.31.0.zip
- 748kB  pyclient-docs-0.31.0.zip
- 200kB  cppclient-examples-0.31.0.zip
- 39MB  javadocs-0.31.0.zip
- 694kB  cppclient-docs-0.31.0.zip

Other Assets
- 182MB  server-jetty-0.30.1.zip
- 194MB  server-jetty-0.30.1.tar
- 98kB  pydeephaven-0.30.1-py3-none-any.whl
- 149kB  deephaven_core-0.30.1-py3-none-any.whl
- 19MB   deephaven-core-0.30.1.zip (source)
- 9.2MB  deephaven-core-0.30.1.tar.gz (source)

Reply:
Okay; 39MB is big but not unreasonable. I don't think the workflow artifacts stick around for an extended period of time (I forget exactly how often we've set them to expire), so I wouldn't be too worried about that right now.